### DISCUSSION TITLE
Some mp4 videos wont play after downloading

### DISCUSSION THREAD

**Discussion by alexb1352** [subhra74/xdm#1037] [General]
Hi. So, Im trying to download some videos and they wont play. With .mkv for example no issues, but mp4 are refusing to launch.
How can I fix it? 
p.s. Another thing, in order not to create too many topics. Im using debian and after removing xdm from the xcfe panel - my wifi icon disappeared as well. They were kinda together, after instalation of xdm I mean, appeared as one button in the pannel. I googled a lot, cant find the solution. I tried to add it back but it doesnt appear in the list of items.

**Comment by roachboy** [comment]
https://jable.tv/
Videos not playing after downloading from this site using 8.0.26 BETA
Using 7.2.11 can play normally

**Reply by alexb1352** [reply]
Didn't work.
What I noticed is that xdm offers youe several files to download, like 3.9MB, 4MB, 3.8MB etc, in my case 7 different files. Are the any videos that xdm can't handle? Cause I got a  feeling that yes.

**Reply by alexb1352** [reply]
Googled a bit the subject, some videos might be protected it says. How can I check if that it's the case? Or by not being able to play it Im already getting the answer?

**Reply by alexb1352** [reply]
Any possible solution in that case apart from screen recording?
